database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -131,7 +152,254 @@ exports.queries = [
+                    }
+                    {
+                      code: "Q13District"
+                      index: 200
+                      title: "District"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: { label: "District/Province" }
+                    }
+                  ]
+                }
+              }
+              {
+                code: "S2"
+                title: "Educational information"
+                index: 1
+                templateElementsUsingId: {
+                  create: [
+                    {
+                      code: "S2Page1"
+                      index: 10
+                      title: "Section 2"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: { title: "## Education history", style: "basic" }
+                    }
+                    {
+                      code: "Q1EducationLevel"
+                      index: 20
+                      title: "Organisation Category"
+                      elementTypePluginCode: "radioChoice"
+                      category: QUESTION
+                      helpText: "TODO: This field should be consider as another  application related to applicant - and simply displayed in company license form"
+                      parameters: {
+                        label: "Education Level"
+                        options: ["Secondary", "University"]
+                        layout: "inline"
+                      }
+                    }
+                    {
+                      code: "Q2Secondary"
+                      index: 30
+                      title: "Secondary"
+                      elementTypePluginCode: "longText"
+                      category: QUESTION
+                      parameters: { label: "Please enter details for secondary"}
+                      visibilityCondition: {
+                        operator: "="
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.Q1EducationLevel.text"]
+                          }
+                          "Secondary"
+                        ]
+                      }
+                    }
+                    {
+                      code: "Q3UniversityHistory"
+                      index: 40
+                      title: "Education List"
+                      elementTypePluginCode: "listBuilder"
+                      category: QUESTION
+                      parameters: {
+                        label: "Education history"
+                        createModalButtonText: "Add to education history"
+                        modalText: "## Education history entry \\n\\nPlease enter details for university"
+                        displayType: "card"
```

Comment:
Would suggest adding a custom `displayFormat` definition (see my one in Company registration if you want a starting point)

Reply:
Actually, `displayType: "table"` might be better for this, perhaps?

Reply:
Yes, good point.

Reply:
Done: 23639b5